test/unit/detect-transaction-example-numbers-test.coffee

```diff
@@ -17,47 +17,21 @@ scenario = (description, {actionContent, examples, exampleNumbersPerTransaction}
       ### Catch a Smurf [POST]
       #{actionContent}
     """
-    action = undefined # API Blueprint AST
-    transition = undefined # API Elements (a.k.a. Refract)
+
+    transitionElements = undefined
+    transactionExampleNumbers = undefined
 
     beforeEach((done) ->
-      options = {type: 'ast', generateSourceMap: true}
-      drafter.parse(apiBlueprint, options, (err, parseResult) ->
-        return done(err) if err
-        action = parseResult.ast.resourceGroups[0].resources[0].actions[0]
-        done()
-      )
-    )
-    beforeEach((done) ->
-      options = {type: 'refract', generateSourceMap: true}
-      drafter.parse(apiBlueprint, options, (err, parseResult) ->
-        return done(err) if err
-        transition = parseResult.content[0].content[0].content[0].content[0]
+      parse(apiBlueprint, (args...) ->
+        [error, {mediaType, apiElements}] = args
+        transitionElements = apiElements.findRecursive('transition').first()
```

Comment:
This line is the biggest performance bottleneck, the following patch will drop nearly 12 seconds off of the test suite.

```diff
-        transitionElements = apiElements.findRecursive('transition').first()
+        transitionElements = apiElements.api.resourceGroups.first().resources.first().transitions.first()
```